src/lints/function_export_name_changed.ron

```diff
@@ -0,0 +1,47 @@
+SemverQuery(
+    id: "function_export_name_changed",
+    human_readable_name: "function's export name has changed or been removed",
+    description: "A function's ABI name with #[no_mangle] or #[export_name = \"name\"] has changed or been removed",
+    required_update: Major,  // Not specified in rust semver doc? most similar is for #[repr] changes: https://doc.rust-lang.org/cargo/reference/semver.html#type-layout
+    reference_link: Some("https://doc.rust-lang.org/reference/abi.html#the-no_mangle-attribute"),  // TODO
```

Comment:
Did you mean to leave both of these comments in the file that will get merged?

Asking because the TODO seems completed, and the other comment seemed like a question for reviewers that I believe I replied to in my previous comment near here.